effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
FYI: the summary that @chatchavan posted isn't an accurate representation of APA standards. Here's the text from the 6th edition:

>  Round as much as possible while keeping prospective use and statistical precision in mind. As a general rule, fewer decimal digits are easier to comprehend than more digits; therefore, in general, it is better to round to two decimal places or to rescale the measurement (in which case effect sizes should be presented in the same metric). For instance, a difference in distances that must be carried to four decimals to be seen when scaled in meters can be more effectively illustrated in millimeters, which would require only a few decimal digits to illustrate the same difference. As a rule, **when properly scaled, most data can be effectively presented with two decimal digits of accuracy**.

I interpret the properly scaled part as basically scientific notation: 1.22 * 10^scale